frontend/Lighting.jsx

```diff
@@ -67,43 +69,78 @@ function Lighting() {
   }), [lightOn, lightColor, lightIntensity]));
 
   return (
-    <View style={styles.container}>
-      <Text>
-        {'Turn Light On\n\n'}
-      </Text>
-      <Switch
-        trackColor={{ false: '00FF00', true: 'FFFFFF' }}
-        onValueChange={(value) => setLightOn(value)}
-        value={lightOn}
-        style={{ transform: [{ scaleX: 2 }, { scaleY: 2 }] }}
-      />
-      <Text>
-        {`\n\n\n\nIntensity of Light: ${Math.round(lightIntensity)}`}
-      </Text>
-      <Slider
-        minimumValue={0}
-        maximumValue={3}
-        disabled={!lightOn}
-        value={lightIntensity}
-        onValueChange={(value) => setLightIntensity(value)}
-        style={{ width: 160, height: 50 }}
-        minimumTrackTintColor="#ffd700"
-        maximumTrackTintColor="#fafad2"
-      />
-      <Text>
+    <LinearGradient
+      colors={['#F9F6FF', '#CFDFF7']}
+      style={{
+        flex: 1,
+        padding: 10,
+        paddingTop: 80,
+        alignItems: 'center',
+      }}
+    >
+      <View style={{ flexDirection: 'row', justifyContent: 'space-between', marginBottom: 30 }}>
+        <LightingTitle>Lighting</LightingTitle>
+        <Switch
+          trackColor={{ false: '00FF00', true: 'FFFFFF' }}
```

Comment:
This line doesn't do anything as the colors need to be in hexadecimal notation. However, I don't think this color scheme change is desirable irregardless.
```suggestion
          trackColor={{ false: '#00FF00', true: '#FFFFFF' }}
```

Reply:
This line is an Android only thing (the prop for iOS is iOS_backgroundColor), and it makes sure that the switch actually appears as a switch, not just a dot. The first two images are without the trackColor prop, and the last image is with it. 
![Screenshot_20210201-140933_Expo Go](https://user-images.githubusercontent.com/73803304/106524377-55f6a980-6497-11eb-92a4-ee9bf682f2e2.jpg)
![Screenshot_20210201-140936_Expo Go](https://user-images.githubusercontent.com/73803304/106524384-5a22c700-6497-11eb-9786-6cdbaa7a747a.jpg)
![Screenshot_20210201-140946_Expo Go](https://user-images.githubusercontent.com/73803304/106524389-5bec8a80-6497-11eb-91f8-15ea276a9f09.jpg)

Reply:
This line is not Android only. As you can see here, correctly specifying the color in hexadecimal format as color references are defined [here](https://reactnative.dev/docs/colors) and results in changes on iOS as well. I'm not saying we should remove this line, but we probably want to tweak the color so that the true value is not just white.

<img width="478" alt="image" src="https://user-images.githubusercontent.com/3270324/106525377-be925600-6498-11eb-8976-f01bf05d8180.png">
<img width="478" alt="image" src="https://user-images.githubusercontent.com/3270324/106525437-d073f900-6498-11eb-99fc-54297f80b023.png">

Reply:
gotcha, my bad. I can adjust it.